### ISSUE TITLE
Scan all installed applications

### ISSUE THREAD

**Issue by Victor239** [jonathanklee/Sapio#6]
It would be great to have a button on the Evaluate page to scan all installed apps rather than just one.

**Comment by jonathanklee** [comment]
@Victor239 Hello. Do you mean being able to select several apps at the same time and provide an evaluation for each of them ?

**Comment by Victor239** [comment]
Yes, but with a "select all" toggle so that all apps can be evaluated in one-click.

**Comment by jonathanklee** [comment]
@Victor239 alright
